packages/truffle-contract/lib/execute.js

```diff
@@ -113,57 +118,63 @@ var execute = {
       return new Promise(async (resolve, reject) => {
         let result;
         try {
-
           await constructor.detectNetwork();
+          args = utils.convertToEthersBN(args);
           result = await fn(...args).call(params, defaultBlock);
-          result = reformat.numbers.call(constructor, result, methodABI.outputs);
+          result = reformat.numbers.call(
+            constructor,
+            result,
+            methodABI.outputs
+          );
           resolve(result);
-
         } catch (err) {
           reject(err);
         }
       });
     };
   },
 
-
   /**
    * Executes method as .send
-   * @param  {Function}   fn       Method to invoke
-   * @param  {String}     address  Deployed address of the targeted instance
+   * @param  {Function} fn         Method to invoke
+   * @param  {Object}   methodABI  Function ABI segment w/ inputs & outputs keys.
+   * @param  {String}   address    Deployed address of the targeted instance
    * @return {PromiEvent}          Resolves a transaction receipt (via the receipt handler)
    */
-  send: function(fn, address) {
+  send: function(fn, methodABI, address) {
```

Comment:
I now pass the `methodABI` through so that it can be used in `utils.getTxParams`